### ISSUE TITLE
pre-built-binary won't be created if xcodeproj does not contain `Release` configuration.

### ISSUE THREAD

**Issue by muukii** [leavez/cocoapods-binary#138]
To solve, add `Release` configuration. even we don't use in mainly. just for creating pre-built-binary.

**Comment by azilbershtein** [comment]
@muukii How can you add it automatically?
What I experienced is for some pods there's Release configuration and some doesn't have and it built the .framework in a wrong location + doesn't have Bundle Id (due to lack of configration).
The fix for me was to change the configuration https://github.com/leavez/cocoapods-binary/blob/master/lib/cocoapods-binary/rome/build_framework.rb#L4 to Debug.
![Screen Shot 2021-01-07 at 18 15 00](https://user-images.githubusercontent.com/64828042/103922152-07feb800-511c-11eb-8716-e7253867c4cd.png)

**Comment by muukii** [comment]
@azilbershtein 

> add Release configuration

Sorry missing words, I mean 'Adds Release configuration into main project (it's not Pods project)`

CocoaPods creates targets according to your project's configurations.
If your project has targets (A,B,C), Pods also would have configurations (A,B,C) each target.

Let me check your current status, does your project have `Release` configuration?

(`your project` means it has Application target.)

**Comment by azilbershtein** [comment]
@muukii No, but as you can see on the left pod it created two configurations:
"Enterprise DEBUG" and "Release" (don't have either of them in my target) and those config doesn't have a bundle Id.
I'm just wondering.. why is that?
It's not something that should be specified in the podspec? (there's nothing about configs there

Thanks for your answer!

**Comment by ben8987** [comment]
We had this issue, and found out that it's due to a CocoaPods issue so raised it there: https://github.com/CocoaPods/CocoaPods/issues/10674
